runtime/drivers/bigquery/arrow.go

```diff
@@ -8,10 +8,10 @@ import (
 	"time"
 
 	"cloud.google.com/go/bigquery"
-	"github.com/apache/arrow/go/v13/arrow"
-	"github.com/apache/arrow/go/v13/arrow/array"
-	"github.com/apache/arrow/go/v13/arrow/ipc"
-	"github.com/apache/arrow/go/v13/arrow/memory"
+	"github.com/apache/arrow/go/v12/arrow"
```

Comment:
Yes, otherwise it leads to
```
panic: proto: file "Flight.proto" is already registered
        previously from: "github.com/apache/arrow/go/v12/arrow/flight/internal/flight"
        currently from:  "github.com/apache/arrow/go/v13/arrow/flight/internal/flight"
See https://protobuf.dev/reference/go/faq#namespace-conflict
```
Snowflake uses v12 and I don't see any PRs or issues in a corresponding repo to move to v13 or even v14.

Reply:
Okay we will need to solve this soon since we need to upgrade to v14. I see this open [issue](https://github.com/apache/arrow/issues/37353) on arrow as well.
I guess for now we can merge this and try to find alternatives once we upgrade to v14 unless you or @begelundmuller have other easy solutions.